docs/components.md

```diff
@@ -21,28 +21,27 @@ KubeVirt consists of a set of services:
 
 ## Example flow: Create and Delete a VM
 
-<!-- FIXME: wrong in a lot of ways and therefore misleading -->
-
-    User       Virt API      VM TPR    Virt Controller    k8s      VM Pod
-    createVM  --> |                           |            |
-                  o create --> +              |            |
-                  |            | <~ create ~> ?            |
-                  |            |              |            |
-                  |            |              o create --> o ------> +
-                  |            |              |            |         |
-                  |            | <---------------------- get VM Spec o
-                  |            o ----------------------------------> |
-                  |            |              |            |         o defineVM()
-                  |            |              |            |         o watchVM()
-                  |            |              |            |         |
-    deleteVM -->  o delete --> *              |            |         |
-                  |              <~ delete ~> ?            |
-                  |                           |            |         |
-                  |                           o delete --> o ------> *
-                  |                           |            |
-                  :                           :            :
-
-    Legend: ?: Event notification
+    Client        Virt API      VM TPR  Virt Controller    k8s      VM Pod
+    ------------- ------------- ------- ------------------ -------- ------
+
+                  listen <------------- WATCH /vms
+                                           |
+    POST /vms ----> validate               |
+                    create ---> VM ---> observe
+                      |         |          v
+                      |         |       POST /pods ----> validate
+                      |         |          |             create --> Pod
+                      |         |          |                         |
+                      |         |          |                         v
+                      | <-----------------------------------------> GET /vms
+                      |         |          |                        defineVM
+                      |         |          |                        watchVM
+                      |         |          |                         |
+    DELETE /vms --> validate    |          |                         |
+                    delete ---> * --------------------------------> observe (handler)
```

Comment:
Sketching this flow becomes ugly.

The flow is not nice ..
client -> api -> k8s
handler -> k8s

This flow is not nice.

To me it look slike we need to clarify the semantics instance vs statu sof instance (i.e. existing VM instance vs status of VM instance).

We need to discuss it I think.

/cc @rmohr @stu-gott